zerver/openapi/zulip.yaml

```
@@ -8797,17 +8797,21 @@ paths:
         - Users can unsubscribe any user from a stream if they [have
           access](/help/stream-permissions) to the stream and are a
           member of the [user group](/api/get-user-groups) specified
-          by the [`can_remove_subscribers_group_id`][can-remove-parameter]
+          by the [`can_remove_subscribers_group`][can-remove-parameter]
           for the stream.
 
-        **Changes**: Before Zulip 7.0 (feature level 161), the
+        **Changes**: In Zulip 8.0 (feature level 192),
+        `can_remove_subscribers_group_id` was renamed to
+        `can_remove_subscribers_group`.
```

Generally we write the "Changes" items to be read going back in time from the present: the reader has started by reading about how this part of the API works for current servers, and then they're building on that to understand how it used to work.

So that produces something like:
```suggestion
        **Changes**: Before Zulip 8.0 (feature level 192),
        the `can_remove_subscribers_group` setting
        was named `can_remove_subscribers_group_id`.
```

(We should really write that down somewhere. I guess https://zulip.readthedocs.io/en/latest/documentation/openapi.html#zulip-swagger-yaml-style is the spot to add it; that page is linked from a comment at the top of this YAML file.)

---

I am not very much familiar with our style of writing documentation and explaining them through examples. So, better to add this tip in separate PR.

---

@laurynmm FYI for the suggested changes to documentation for API docs.